integration/hurl/tests_failed/unix_socket.err

```diff
@@ -0,0 +1,7 @@
+error: HTTP connection
+  --> tests_ok/unix_socket.hurl:1:5
+   |
+ 1 | GET http://example/hello
+   |     ^^^^^^^^^^^^^^^^^^^^ (7) Failed to connect to example port 80 after 0 ms: Couldn't connect to server
```

Comment:
So libcurl version has not the same exact message (that's why this test ik KO on the CI for some Linux version). When we've this kink of issue, we "patternize" the err file:

1. change filename from `unix_socket.err` to `unix_socker.err.pattern`
2. replace any variant with `~` (it will skipped when diff is done)
   
Instead of:

```
1 | GET http://example/hello
    |     ^^^^^^^^^^^^^^^^^^^^ (7) Failed to connect to example port 80 after 0 ms: Couldn't connect to server
```

We can write:

```
1 | GET http://example/hello
    |     ^^^^^^^^^^^^^^^^^^^^ (7) ~~~
```

It will be sufficient (we test the libcurl error return rather than the message)

Reply:
Nice! That's very nifty.